doc/classes/FileDialog.xml

```diff
@@ -116,10 +118,11 @@
 	<members>
 		<member name="access" type="int" setter="set_access" getter="get_access" enum="FileDialog.Access" default="0">
 			The file system access scope. See [enum Access] constants.
-			[b]Warning:[/b] Currently, in sandboxed environments such as Web builds or sandboxed macOS apps, FileDialog cannot access the host file system. See [url=https://github.com/godotengine/godot-proposals/issues/1123]godot-proposals#1123[/url].
+			[b]Warning:[/b] In Web builds, FileDialog cannot access the host file system. In sandboxed Linux and macOS environments, [member use_native_dialog] is automatically used to allow limited access to host file system.
 		</member>
 		<member name="current_dir" type="String" setter="set_current_dir" getter="get_current_dir">
 			The current working directory of the file dialog.
+			[b]Note:[/b] This property might be ignored by native file dialogs.
```

Comment:
This note is way too broad and, from the reader's point of view, it's meaningless. If anything, it implies the property is unreliable and you shouldn't use it.

Reply:
> it implies the property is unreliable and you shouldn't use it.

It is unreliable.

For the reference, description for the underlying native implementation docs:

> Suggested folder from which the files should be opened.
> The portal implementation is free to ignore this option.

Reply:
It's worth briefly explaining *why* that's the case, then, instead of leaving the reader on its own thoughts.